styleguide/overlay.scss

```diff
@@ -11,9 +11,11 @@ $overlay-margin: 10vh;
   height: 100%;
   left: 0;
   opacity: 0;
-  overflow: scroll;
+  overflow: auto;
```

Comment:
overflow: scroll isn't needed anymore for mobile scrolling, but _does_ force scrollbars in certain situations. auto + `-webkit-overflow-scrolling: touch` is the current best practice for making momentum scrolling happen.